backend/src/routes/auth.route.ts

```diff
@@ -0,0 +1,9 @@
+import express from 'express';
+const router = express.Router();
+import { register, login } from '@controllers/auth.controller';
+import { authMiddleware } from '@middlewares/auth.middleware';
+
+router.post('/register', register);
+router.post('/login', login);
```

Comment:
```suggestion
router.post('/login', authMiddleware, login);
```

Apply the middleware here so it'll be executed before the controller.